test/conversion.spec.ts

```diff
@@ -0,0 +1,98 @@
+import 'mocha';
+import { expect } from 'chai';
+import { MassaAmount, MASSA_UNIT } from '../src';
+
+describe('MassaAmount Class', () => {
+  it('classical unit test conversion cases', () => {
+    const amount = new MassaAmount(1, MASSA_UNIT.MASSA);
+    expect(amount.toString()).to.equal('1 MASSA');
+
+    const milliMassa = amount.toUnit(MASSA_UNIT.MILLI_MASSA);
+    expect(milliMassa.toString()).to.equal('1000 MILLI_MASSA');
```

Comment:
I would have define a custom toString function to have more friendly currency name:
- MILLI_MASSA => mMASSA
- MICRO_MASSA => µMASSA
- NANO_MASSA => nMASSA